schemas/blocks/definitions.json

```diff
@@ -14,6 +14,9 @@
       },
       "contents": {
         "$ref": "https://eq.ons.gov.uk/common_definitions.json#/contents"
+      },
+      "interviewer_only": {
+        "$ref": "https://eq.ons.gov.uk/common_definitions.json#/interviewer_only"
```

Comment:
Not sure why this is here?. This should only be needed for questions and interstitials not all block types.

Reply:
It's only for question and interstitial page, how do I make it interstitial specific if it uses `content` key with all sub definitions in `schemas/blocks/definitions.json`?

Reply:
Should be able to add it to `interstitial.json`

Reply:
As discussed, we should add this to the block level for interstitial/question blocks.

Reply:
Moved it to block level, thanks👍